experiments/sidb_simulation/runtime_analysis.cpp

```diff
@@ -0,0 +1,114 @@
+//
+// Created by Jan Drewniok on 17.05.24.
+//
+
+#include "fiction/algorithms/iter/bdl_input_iterator.hpp"
+#include "fiction/algorithms/simulation/sidb/exhaustive_ground_state_simulation.hpp"
+#include "fiction/algorithms/simulation/sidb/quickexact.hpp"
+#include "fiction/algorithms/simulation/sidb/quicksim.hpp"
+#include "fiction/algorithms/simulation/sidb/sidb_simulation_parameters.hpp"
+#include "fiction/algorithms/simulation/sidb/time_to_solution.hpp"
+#include "fiction/io/read_sqd_layout.hpp"
+#include "fiction/layouts/coordinates.hpp"
+#include "fiction/types.hpp"
+#include "fiction/utils/truth_table_utils.hpp"
+#include "fiction_experiments.hpp"
+#include "mockturtle/utils/stopwatch.hpp"
+
+#include <fmt/format.h>
+
+#include <array>
+#include <filesystem>
+#include <string>
+#include <utility>
+#include <vector>
+
+using namespace fiction;
+
+int main()  // NOLINT
+{
+    experiments::experiment<std::string, std::size_t, double, double, double, double, double> simulation_exp{
+        "Benchmark",
+        "Gate Name",
+        "#Instances",
+        "Exhaustive Runtime in s",
+        "QuickExact Runtime in s",
+        "Average QuickSim Accuracy of all instances",
+        "QuickSim Single Runtime in s",
```

Comment:
```suggestion
        "Exhaustive Runtime [s]",
        "QuickExact Runtime [s]",
        "Average QuickSim Accuracy",
        "QuickSim Single Runtime [s]",
```

Reply:
@marcelwa I see what you mean, but then we are inconsistent regarding ``operational_domain_siqad.cpp``.